### PR TITLE
[DBAAS-1251] Code cleanup for linter issues

### PR DESCRIPTION
**Pull request by jianrongzhang89**
## Description
Clean up code found by the linter with 'make lint' command. See the list of issues attached.
-- Updated comments
-- renamed offending variables
-- removed unused variables

Such cleanup should not impact the functionality of the operator.
[lint-issues.txt](https://github.com/RHEcosystemAppEng/dbaas-operator/files/10911919/lint-issues.txt)

Jira ticket: https://issues.redhat.com/browse/DBAAS-1251

## Type of change

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

## Checklist (Definition of Done)

- [ ] All acceptance criteria specified in JIRA or in issue number have been completed
- [ ] Unit tests added that prove the fix is effective or the feature works 
- [ ] Documentation added for the feature
- [ ] all relevant tests are passing
- [X ] Code Review completed
- [ ] Verified independently by reviewer

### CODE REVIEW

File: api/v1alpha1/dbaasinstance_conversion.go [hunk 1]
```diff
@@ -26,60 +26,60 @@ import (
 )
 
 // ConvertTo converts this DBaaSInstance to the Hub version (v1beta1).
-func (src *DBaaSInstance) ConvertTo(dstRaw conversion.Hub) error {
+func (c *DBaaSInstance) ConvertTo(dstRaw conversion.Hub) error {
```
Comment (tchughesiv) [inline api/v1alpha1/dbaasinstance_conversion.go:29]: why `c` for all of these?

Comment (jianrongzhang89) [reply api/v1alpha1/dbaasinstance_conversion.go:29]: "c" is an abbreviation for  "current object". 

The linter complains:
receiver name dst should be consistent with previous receiver name src for DBaaSInstance.

Reference:
https://github.com/golang/go/wiki/CodeReviewComments#receiver-names

Comment (tchughesiv) [reply api/v1alpha1/dbaasinstance_conversion.go:29]: per that reference...

_The name of a method's receiver should be a reflection of its identity; often a one or two letter abbreviation of its type suffices (**such as "c"** or "cl" for "Client")_

The `c` here appears to be an example of what one _might_ use.
Also, the existing code was done according to k8s examples of conversion code.
https://book.kubebuilder.io/multiversion-tutorial/conversion.html

Comment (tchughesiv) [reply api/v1alpha1/dbaasinstance_conversion.go:29]: I'd prefer to keep the k8s standard of `src` here, as it matches with the existing `dst`... also used here.

Comment (tchughesiv) [reply api/v1alpha1/dbaasinstance_conversion.go:29]: idk, thoughts @redhatHameed ?

Comment (redhatHameed) [reply api/v1alpha1/dbaasinstance_conversion.go:29]: @tchughesiv @jianrongzhang89 I prefer also to keep 'src' as the k8s standard.

**Review by tchughesiv — DISMISSED** [review]
lgtm

File: config.toml [hunk 2]
```diff
@@ -17,7 +17,6 @@ warningCode = 0
 [rule.var-declaration]
 [rule.package-comments]
 [rule.range]
-[rule.receiver-naming]
```
Comment (tchughesiv) [inline config.toml:20]: @jianrongzhang89 why are we removing this?

Comment (jianrongzhang89) [reply config.toml:20]: This allows the linter to skip reporting the receiver issues we discussed above for those conversion functions.

**Review by tchughesiv — APPROVED** [review]
lgtm

**Pull request merged**